test/suites/clustering.sh

```diff
@@ -2257,13 +2257,20 @@ test_clustering_image_refresh() {
   LXD_DIR="${LXD_REMOTE_DIR}" lxc publish c1 --alias testimage --public
   new_fingerprint="$(LXD_DIR="${LXD_REMOTE_DIR}" lxc image ls testimage -c f --format csv)"
 
+  pids=""
+
   # Trigger image refresh on all nodes
   for lxd_dir in "${LXD_ONE_DIR}" "${LXD_TWO_DIR}" "${LXD_THREE_DIR}"; do
     LXD_DIR="${lxd_dir}" lxc query /internal/image-refresh &
+    pids="$! ${pids}"
   done
 
-  # Wait for the image to be refresh
-  sleep 5
+    # Wait for the image to be refreshed
```

Comment:
Is it just github or is this comment badly indented?

Reply:
It was just badly indented.